
Watching for file changes with StatReloader
/home/asadbek/Desktop/projects/myenvs/lib/python3.12/site-packages/pydantic/_internal/_config.py:345: UserWarning: Valid config keys have changed in V2:
* 'orm_mode' has been renamed to 'from_attributes'
  warnings.warn(message, UserWarning)
Performing system checks...

System check identified no issues (0 silenced).
February 25, 2025 - 20:03:24
Django version 5.1.4, using settings 'app.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.

Watching for file changes with StatReloader
/home/asadbek/Desktop/projects/myenvs/lib/python3.12/site-packages/pydantic/_internal/_config.py:345: UserWarning: Valid config keys have changed in V2:
* 'orm_mode' has been renamed to 'from_attributes'
  warnings.warn(message, UserWarning)
[25/Feb/2025 20:04:29] "GET /admin/ HTTP/1.1" 200 32639
[25/Feb/2025 20:05:00] "GET /admin/Certificate/certificate/ HTTP/1.1" 200 28122
[25/Feb/2025 20:05:00] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
[25/Feb/2025 20:05:01] "GET /media/certificates/qrcodes/qr_CR-1-1-20250225.png HTTP/1.1" 304 0
[25/Feb/2025 20:05:03] "GET /admin/Certificate/certificate/2/change/ HTTP/1.1" 200 30316
[25/Feb/2025 20:05:03] "GET /admin/jsi18n/ HTTP/1.1" 200 3342
Performing system checks...

System check identified no issues (0 silenced).
February 25, 2025 - 20:04:24
Django version 5.1.4, using settings 'app.settings'
Starting development server at http://127.0.0.1:8000/
Quit the server with CONTROL-C.

